.github/workflows/release_otelbench.yaml

```diff
@@ -0,0 +1,106 @@
+run-name: release-otelbench
+name: Release otelbench docker image on elastic container library
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version of the otelbench image to release, should follow format "vN.N.N".
+        required: true
+  push:
+    branches:
+      - main
+    paths:
+      - ./loadgen/cmd/otelbench/Makefile
+
+jobs:
+  check-release:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get-new-version.outputs.new_version }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
+      - name: Checks if release is required.
+        id: get-new-version
+        run: ./scripts/check_release.sh "${{ github.event_name }}" "${{ github.event.inputs.version }}"
```

Comment:
maybe you need `continue-on-error: true`? Otherwise, this will fail and might confuse what's the rationale.

OTOH, I wonder whether this approach can be changed to do something like:
- https://github.com/elastic/apm-aggregation-service/blob/52b0be77db5049433fb9ef783cbc4bf0cff1d75d/.github/workflows/create-release-tag.yml#L26-L45

and then you use https://github.com/elastic/apm-aggregation-service/blob/52b0be77db5049433fb9ef783cbc4bf0cff1d75d/.github/workflows/create-release-tag.yml#L57-L65 in the `release-otelbench:` job.


The idea is to rely on the GH releases to create new releases if no release was done.

Reply:
Sure, but it will be shown as a failure in https://github.com/elastic/opentelemetry-collector-components/actions, and the failure can be for anything.

Reply:
Fair enough